src/app.rs

```diff
@@ -322,18 +321,43 @@ impl Application {
                 #[cfg(feature = "api")]
                 api_server,
             };
+            let topology_controller = Arc::new(Mutex::new(topology_controller));
+
+            // If the relevant ENV var is set, start up the control server
+            #[cfg(not(windows))]
```

Comment:
I understood Windows also supported named sockets. Does this need to be guarded this way?

Reply:
That support is not yet reflected in common libraries, so supporting Windows would require a bit of extra manual plumbing on our part. Given that we do not support Windows for the first client of this API, I've opted to defer that work until there is a need.